addons/l10n_lu/i18n_extra/fr.po

```diff
@@ -2057,6 +2057,11 @@ msgstr "Virements internes : solde créditeur"
 msgid "Internal transfers : debit balance"
 msgstr "Virements internes : solde débiteur"
 
+#. module: l10n_lu
+#: model:account.fiscal.position.template,name:l10n_lu.account_fiscal_position_template_private_LU_IC
+msgid "EU private"
+msgstr ""
```

Comment:
@rgo-odoo a bit weird to add translation but not the translation itself.  EU privé (fr)  EU privat (de)